docs/Getting_Involved/Testing.rst

```diff
@@ -9,8 +9,15 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following
+the `coala Tutorial <Users/Tutorials/Tutorial.html>`_.
+
+Before you run tests, make sure you
+`install coala
+<Users/Install.html/>`_.
```

Comment:
please wrap this into the previous line :+1:, and if possible even in the line before the previous one